docs/reference/configuration.rst

```diff
@@ -1014,7 +1014,8 @@ CMS_TOOLBAR_HIDE
 default
     ``False``
 
-If True, the toolbar is hidden in the pages out django CMS.
+If True, the toolbar is not displayed in pages that use the ``{% cms_toolbar %}`` template tag if they are not
+actually being handled by a django CMS view.
```

Comment:
The we end up with weird things like "non-django CMS" (*non-django*? eh? which is simply incorrect) or *non-django-CMS* (which is unaesthetic).